core/storage/statistics/gae_models.py

```diff
@@ -107,6 +116,16 @@ class StartExplorationEventLogEntryModel(base_models.BaseModel):
 
     @classmethod
     def get_new_event_entity_id(cls, exp_id, session_id):
+        """Generates entity ID for a new event based on its
+        exploration and session id.
+
+        Args:
+            exp_id: str.
```

Comment:
Add an explanation for the arg like `str. ID of the exploration` here and everywhere else.

Reply:
done..